# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Fri Jan 27 19:06:16 2023 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Fri Jan 27 21:11:42 2023 +0200
@@ -7039,19 +7039,6 @@
   value: true
   mirror: always
 
-# Whether to allow decoding of animated AVIF sequences.
-- name: image.avif.sequence.enabled
-  type: RelaxedAtomicBool
-  value: false
-  mirror: always
-
-# Whether AVIF files containing sequences should be animated even when the
-# major brand is set to 'avif'.
-- name: image.avif.sequence.animate_avif_major_branded_images
-  type: RelaxedAtomicBool
-  value: false
-  mirror: always
-
 # Whether we attempt to decode JXL images or not.
 - name: image.jxl.enabled
   type: RelaxedAtomicBool